python_modules/libraries/dagster-dbt/dagster_dbt/cloud/cli.py

```diff
@@ -67,7 +67,7 @@ def cache_compile_references(
             job_id=job_id,
             cause="Generating software-defined assets for Dagster.",
             steps_override=[dbt_compile_command],
-            generate_docs_override=True,
+            generate_docs_override=dbt_cloud_job.get("generate_docs", False),
```

Comment:
can this just be `None` (aka removing this line entirely)? This parameter is an override, so wouldn't not setting it accomplish the goal of inheriting the `generate_docs` setting from the main job?